### ISSUE TITLE
Uncaught Error: Cannot use object of type stdClass as array

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#5333]
Hello,
my PHP error log is flooded with messages about two uncaught errors - one in mod/photos.php and one in mod/fetch.php.
Also my workerqueue exploded over night (0 -> 5000).
I'm currently pulling the latest updates, but now the database structure is updated........ :-)
What a morning! ;-) :coffee: 

````
[08-Jul-2018 08:26:03 UTC] PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array in /var/www/html/libranet/mod/fetch.php:57
Stack trace:
#0 /var/www/html/libranet/index.php(365): fetch_init(Object(Friendica\App))
#1 {main}
  thrown in /var/www/html/libranet/mod/fetch.php on line 57
[08-Jul-2018 08:26:15 UTC] PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array in /var/www/html/libranet/mod/photos.php:89
Stack trace:
#0 /var/www/html/libranet/index.php(365): photos_init(Object(Friendica\App))
#1 {main}
  thrown in /var/www/html/libranet/mod/photos.php on line 89
[08-Jul-2018 08:26:22 UTC] PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array in /var/www/html/libranet/mod/fetch.php:57
Stack trace:
#0 /var/www/html/libranet/index.php(365): fetch_init(Object(Friendica\App))
#1 {main}
  thrown in /var/www/html/libranet/mod/fetch.php on line 57
````

### Friendica version you encountered the problem
2018.08-dev (last pull yesterday evening)

### Friendica source (git, zip)
git

### PHP version
7.0

### SQL version
10.0 MariaDB

**Comment by annando** [comment]
Yeah, I'm currently fixing all these warnings. The PHP settings had been changed so that warnings are now displayed as well - but now many places have to be fixed.

**Comment by AlfredSK** [comment]
And more of this...

````
[08-Jul-2018 09:35:34 UTC] PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array in /var/www/html/mordor/src/Model/Contact.php:818
Stack trace:
#0 /var/www/html/mordor/mod/item.php(598): Friendica\Model\Contact::getIdForURL('https://forum.f...')
#1 /var/www/html/mordor/index.php(379): item_post(Object(Friendica\App))
#2 {main}
  thrown in /var/www/html/mordor/src/Model/Contact.php on line 818
````

By the way. This is breaking Friendica at the moment. I get a lot of whitescreens. I cannot post something. I get a WSOD after clicking on "share". Also, the post preview in frio isn't working.
What I want to say is: I have the impression this is not just about warnings - something is really broken at the moment.

**Comment by AlfredSK** [comment]
Hmm. The WSODs (error 500) are gone after disabling redis caching.

**Comment by MrPetovan** [comment]
>Yeah, I'm currently fixing all these warnings. The PHP settings had been changed so that warnings are now displayed as well - but now many places have to be fixed.

This error message has nothing to do with re-enabling the notice messages. See #5330 instead for newly displaying Notice messages.

> Hmm. The WSODs (error 500) are gone after disabling redis caching.

I'd guess this is caused by the `json_decode` in the Redis Cache driver returning an object instead of an expected array:
https://github.com/friendica/friendica/blob/c6efbcd1aad965aebba1533bdeca26a66ae7339d/src/Core/Cache/RedisCacheDriver.php#L33-L53

**Comment by MrPetovan** [comment]
Closed via #5339
